SUMMARY.md

```diff
@@ -415,10 +415,7 @@
 
 # Security
 
-- [Security](./doc/security/README.md)
-- [Implementation Guidelines](./doc/security/implementation_guidelines/README.md)
```

Comment:
Where did this link end up?

Reply:
I moved Security further up the SUMMARY.md, and renamed the first item to [Security in OpenTitan]